config/application.rb

```diff
@@ -38,7 +38,11 @@ class Application < Rails::Application
 
     config.middleware.insert_before 0, Rack::Cors do
       allow do
-        origins '*'
+        origins(lambda do |source, _env|
```

Comment:
should we add a `TODO: use * after we remove basic auth` ?

Reply:
I'm not a big fan of TODOs. I'd rather open a GitHub issue.

Reply:
works for me!